README.md

```diff
@@ -30,13 +30,14 @@ yarn start
-| PORT               | Where the HTTP server should listen.                                                                                                                                       | No        | `8080`  |
-| STREAMLABS_TOKEN   | A token to use the StreamLabs API. You can get one by using [StreamDevs/streamlabs-token](https://github.com/streamdevs/streamlabs-token)                                  | **Yes**   | _empty_ |
-| TWITCH_BOT_NAME    | The account (username) that the chatbot uses to send chat messages.                                                                                                        | **Yes**   | _empty_ |
-| TWITCH_BOT_TOKEN   | The token to authenticate your chatbot. Generate this with https://twitchapps.com/tmi/, while logged in to your chatbot account. The token will be an alphanumeric string. | **Yes**   | _empty_ |
-| TWITCH_BOT_CHANNEL | The Twitch channel name where you want to run the bot. Usually this is your main Twitch account.                                                                           | **Yes**   | _empty_ |
+| Variable                  | Setting                                                                                                                                                                    | Mandatory | Default       |
+| ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------- | ------------- |
+| PORT                      | Where the HTTP server should listen.                                                                                                                                       | No        | `8080`        |
+| STREAMLABS_TOKEN          | A token to use the StreamLabs API. You can get one by using [StreamDevs/streamlabs-token](https://github.com/streamdevs/streamlabs-token)                                  | **Yes**   | _empty_       |
+| TWITCH_BOT_NAME           | The account (username) that the chatbot uses to send chat messages.                                                                                                        | **Yes**   | _empty_       |
+| TWITCH_BOT_TOKEN          | The token to authenticate your chatbot. Generate this with https://twitchapps.com/tmi/, while logged in to your chatbot account. The token will be an alphanumeric string. | **Yes**   | _empty_       |
+| TWITCH_BOT_CHANNEL        | The Twitch channel name where you want to run the bot. Usually this is your main Twitch account.                                                                           | **Yes**   | _empty_       |
+| NOTIFY_ASSIGNED_ISSUES_TO | A comma-separated list of GitHub user names. Only issues assigned to these users will be notified or leave it empty to receive all notifications.                          | **No**    | _empty array_ |
```

Comment:
Issues and Assigned are flipped here.
The right order would be NOTIFY_ISSUES_ASSIGNED_TO